### ISSUE TITLE
Incorrectly claims to be compatible with Python 3.4

### ISSUE THREAD

**Issue by garrettr** [cipriantarta/django-logging#2]
According to your setup.py, django-logging is compatible with Python 3.4 and 3.5. However, it fails on Python 3.4 due to the incorrect assumption that `traceback.TracebackException` is [available](https://github.com/cipriantarta/django-logging/blob/78a2a30f856ba8aab450a08500993727571d48d7/django_logging/log_object.py#L125-L128)—according to the official [docs](https://docs.python.org/3/library/traceback.html#tracebackexception-objects), this object was only added in 3.5. Thus, running django-logging with Python 3.4 results in numerous exceptions that look like this:

```
Dec 12 16:04:50 hostname gunicorn[20347]: During handling of the above exception, another exception occurred:
Dec 12 16:04:50 hostname gunicorn[20347]: Traceback (most recent call last):
Dec 12 16:04:50 hostname gunicorn[20347]: File "/usr/lib/python3.4/logging/handlers.py", line 71, in emit
Dec 12 16:04:50 hostname gunicorn[20347]: if self.shouldRollover(record):
Dec 12 16:04:50 hostname gunicorn[20347]: File "/usr/lib/python3.4/logging/handlers.py", line 187, in shouldRollover
Dec 12 16:04:50 hostname gunicorn[20347]: msg = "%s\n" % self.format(record)
Dec 12 16:04:50 hostname gunicorn[20347]: File "/home/gcorn/freedompress/lib/python3.4/site-packages/django_logging/handlers.py", line 46, in format
Dec 12 16:04:50 hostname gunicorn[20347]: message = record.msg.to_dict
Dec 12 16:04:50 hostname gunicorn[20347]: File "/home/gcorn/freedompress/lib/python3.4/site-packages/django_logging/log_object.py", line 109, in to_dict
Dec 12 16:04:50 hostname gunicorn[20347]: exception=ErrorLogObject.format_exception(self.exception)
Dec 12 16:04:50 hostname gunicorn[20347]: File "/home/gcorn/freedompress/lib/python3.4/site-packages/django_logging/log_object.py", line 128, in format_exception
Dec 12 16:04:50 hostname gunicorn[20347]: _traceback = traceback.TracebackException.from_exception(exception).exc_traceback
Dec 12 16:04:50 hostname gunicorn[20347]: AttributeError: 'module' object has no attribute 'TracebackException'
```

**Comment by cipriantarta** [comment]
I'll have a look at this, thanks for the issue.

**Comment by cipriantarta** [comment]
fixed in 1.5.4
